cookbook/Berksfile

```diff
@@ -0,0 +1,8 @@
+source 'https://supermarket.chef.io'
+
+metadata
+
+cookbook 'java'
+cookbook 'et_nginx'
```

Comment:
@ssalinas basically, the community nginx cookbook moves rather slowly, and has a number of bugfixes & improvements from us that have yet to land upstream.